components/automate-ui/e2e/app.e2e-spec.ts

```diff
@@ -3,11 +3,11 @@ import { fakeServer } from './helpers/fake_server';
 import { enableWelcomeModal, disableWelcomeModal } from './helpers/welcome_modal';
 
 describe('App', () => {
-  it('lands on the event feed page', () => {
+  it('starts on the top-level domain (before permission-based routing occurs)', () => {
     browser.waitForAngularEnabled(false);
     browser.get('/');
 
-    expect(browser.getCurrentUrl()).toMatch(/\/event-feed$/);
+    expect(browser.getCurrentUrl()).toMatch(/http:\/\/localhost:\d+\//);
```

Comment:
I take it we don't know the port. However, can we somehow say where this ends? I.e. add a `$` anchor?